marmoset/dhcp/isc_dhcp_ldap_config.py

```diff
@@ -29,49 +29,67 @@ def __get_server_connection():
     def save(self):
         conn = self.__get_server_connection()
 
-        dhcpStatements = ["fixed-address %s;" % self.dhcp_config.ip_address,
```

Comment:
i commited something of both worlds :D PEP8 says max line length is 80 chars (IMHO 80 chars as guideline, should be deprecated)

with the commit c5f0698cb377c9824f2d1cb7bde6282b24fa88ab
it's under 80 chars but more readable